io/src/main/java/com/indeed/util/io/Files.java

```diff
@@ -322,19 +333,27 @@ public void writeAndFlushData(@Nonnull OutputStream outputStream) throws IOExcep
         }
     }
 
+    /**
+     * @deprecated Use {@link #writeObjectIfChangedOrDie2(java.lang.Object, java.lang.String)}
+     */
+    @Deprecated
+    public static boolean writeObjectIfChangedOrDie(@Nonnull final Object obj, @Nonnull final String file, @Nonnull final org.apache.log4j.Logger log) throws IOException {
+        return writeObjectIfChangedOrDie2(obj, file);
+    }
+
     public static long computeFileChecksum(@Nonnull final File file, @Nonnull final Checksum checksum) throws IOException {
         return com.google.common.io.Files.asByteSource(file).hash(Hashing.crc32()).padToLong();
     }
 
     /**
      * Writes an object to a file.
      * @return true if the file was successfully written, false otherwise
-     * @deprecated use {@link #writeObjectToFileOrDie(Object, String, org.apache.log4j.Logger)} instead
+     * @deprecated use {@link #writeObjectToFileOrDie2(Object, String)} instead

```

Comment:
I would be ok with simply removing the method that had been previously deprecated.